tests/unit/components/sl-input-test.js

```diff
@@ -0,0 +1,210 @@
+import ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-input', 'Unit | Component | sl input', {
+    unit: true
+});
+
+
```

Comment:
As discussed, removing the enter action as it's not used or referenced in the docs.